libs/application/ui-shell/src/lib/FormShell.tsx

```diff
@@ -105,57 +104,7 @@ export const FormShell: FC<{
                 height="full"
                 borderRadius="large"
                 background="white"
-              >
-                <Sentry.ErrorBoundary
-                  beforeCapture={(scope) => {
-                    scope.setTag('errorBoundaryLocation', 'FormShell')
-                    scope.setExtra('applicationType', application.typeId)
-                    scope.setExtra('applicationState', application.state)
-                    scope.setExtra('currentScreen', currentScreen.id)
-                  }}
-                  fallback={
-                    <ErrorShell
-                      title={formatMessage(coreMessages.globalErrorTitle)}
-                      subTitle={formatMessage(coreMessages.globalErrorMessage)}
-                    />
-                  }
-                >
-                  <Screen
```

Comment:
We are going to need this screen element to display the application form. We just need to remove `<Sentry.ErrorBoundary>` that is wrapped around it.

Reply:
Re-added wrapped code